src/object_functionality.js

```diff
@@ -0,0 +1,54 @@
+//Creating a car class
+class car {
+  constructor(manufacturer, model, year, type, price) {
+    this.manufacturer = manufacturer;
+    this.model = model;
+    this.year = year;
+    this.type = type;
+    this.price = price;
+  }
+
+  toString() {
+    return `
+      Manufacturer: ${this.manufacturer}
+      Model: ${this.model}
+      Year: ${this.year}
+      Type: ${this.type}
+      Price: ${this.price}
+      `;
+  }
```

Comment:
It'd be great to give this car some behaviours other than printing itself, in order to show methods at work. For example. you could have a `drive` method that takes a parameter of km or miles and updates an internal property of `mileage` of the car.